source/text/label.ts

```diff
@@ -320,19 +200,19 @@ export class Label {
      * screen space, world space, ...). This can be set either explicitly or implicitly using various transformation
      * utility functions.
      */
-    set transform(transform: mat4) {
-        if (mat4.equals(this._transform, transform)) {
+    set staticTransform(transform: mat4) {
```

Comment:
add in doc: why is it called `static`? Can it be updated by the user/renderer?

Reply:
next commit will show that i already started on  separating transforms in dynamic and static transforms. A label has to be specified as dynamic or static on construction. When static it behaves as is currently implemented, i.e., all transformations are baked into the glyph vertices. Dynamic labels will allow to only bake the minimal required transforms and allows to update position, rotation, etc. without the need of re-computation or re-typesetting. This, however, will be a little bit slower while rendering.